content/documentation/pages/5-feature-guides/3-batch/1-deployment-properties.md

```diff
@@ -6,12 +6,18 @@ description: 'Initiate a Batch deployment with deployment property overrides'
 
 # Deployment Properties
 
-When task definitions are launched to the target platforms `Local`, `CloudFoundry` and `Kubernetes`, you can provide the configuration properties that are applied to the task applications at launch time.
+When task definitions are launched to the target platforms `local`, `cloudFoundry` and `kubernetes`, you can provide the configuration properties that are applied to the task applications at launch time.
```

Comment:
Change this sentence to:

When task definitions are launched to the target platforms (`local`, `cloudFoundry`, and `kubernetes`), you can provide the configuration properties that are applied to the task applications at launch time.

(The difference is the parentheses around the list and the serial comma in the list.)